purchase_packaging_report/report/purchase_order_templates.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_purchaseorder_document"
+        inherit_id="purchase.report_purchaseorder_document"
+    >
+        <xpath expr="//td/span[@t-field='line.product_qty']/.." position="inside">
+            <div
+                t-if="line.product_packaging_id"
+                groups="product.group_stock_packaging"
+                class="text-muted d-inline-block"
+            >
+                <span>(</span>
+                <span t-field="line.product_packaging_qty" />
+                <span class="mx-0">x</span>
+                <span t-field="line.product_packaging_id.name" />
+                <span>)</span>
```

Comment:
I think https://github.com/OCA/purchase-reporting/pull/67/files#r1620359417 is a valid point.

However, if you still want to keep it as it is here, I'd do a little change to avoid adding extra whitespace around those parenthesis and use the canonical UTF-8 character for multiplication:

```suggestion
                (<span t-field="line.product_packaging_qty" />
                <span class="mx-0">✕</span>
                <span t-field="line.product_packaging_id.name" />)
```

Reply:
Changed to maintain the same appearance as sale_packaging_report